package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "auth-i",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon index.js"
```

Comment:
index.js is not required, in case you did not know. There is no harm in keeping it there though :)